backend/index.js

```diff
@@ -19,7 +18,6 @@ mongoose
     })
 
 // User parsing middleware
-app.use(bodyParser.json())
```

Comment:
Recommend we replace with this to perform the same function.
```
app.use(express.urlencoded({extended: true}));
app.use(express.json()) // To parse the incoming requests with JSON payloads
```
https://stackoverflow.com/questions/66525078/bodyparser-is-deprecated